pkgs/by-name/ze/zed-discord-presence/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "zed-discord-presence";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "xhyrom";
+    repo = "zed-discord-presence";
+    tag = "v${version}";
```

finalAttrs makes overriding easier
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
  pname = "zed-discord-presence";
  version = "0.7.0";

  src = fetchFromGitHub {
    owner = "xhyrom";
    repo = "zed-discord-presence";
    tag = "v${finalAttrs.version}";
```